cl/api/tasks.py

```diff
@@ -201,8 +192,16 @@ def write_json_to_disk(
                 # A non-jurisdiction-centric object.
                 loc = join(bulk_dir, obj_type_str, f"{item.pk}.json")
 
-            with open(loc, "wb") as f:
-                f.write(json_str)
+            try:
+                with open(loc, "wb") as f:
+                    f.write(json_str)
+            except FileNotFoundError:
+                logging.warning("Court directory not found, adding it now.")
+                # If a court is created during the bulk generation, generate the directory for it.
```

Comment:
```suggestion
                logging.warning("Bulk data directory not found, adding it now.")
                # If we have new courts or object types since last generation, 
                # make the needed directories
```